src/App.js

```diff
@@ -1,26 +1,93 @@
-import React from 'react';
-import logo from './logo.svg';
+import React from "react";
+import Dropdown from "react-bootstrap/Dropdown";
+import BookList from "./book/BookList"
+import {
+    BrowserRouter as Router,
+    Link as Link,
+    Switch,
+    Route,
+} from "react-router-dom";
+
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default  function App() {
+    const levels = {
+        "lycée": [
+            {
+                "id": 287,
+                "name": "Terminale"
+            },
+            {
+                "id": 286,
+                "name": "1re"
+            },
+            {
+                "id": 285,
+                "name": "2de"
+            }
+        ],
+        "collège": [
+            {
+                "id": 284,
+                "name": "3ème"
+            },
+            {
+                "id": 282,
+                "name": "4ème"
+            },
+            {
+                "id": 281,
+                "name": "5ème"
+            },
+            {
+                "id": 283,
+                "name": "6ème"
+            }
+        ]
+    }
+
+    return (
+        <Router>
+            <header className="App App-header">
+                <Link to="/books" className="btn btn-basic">Accueil</Link>
+                {Object.keys(levels).map(key=>{
+                    console.log('App levels.map :', key)
+                    return (
+                        <Dropdown key={`dropdown-${key}`}>
+                            <Dropdown.Toggle id={`dropdown-${key}`} variant="basic">
+                                {key}
+                            </Dropdown.Toggle>
+                            <Dropdown.Menu>
+                                <Dropdown.Item href={`/books/${key}/`}>Tous les niveaux</Dropdown.Item>
+                                {levels[key].map(
+                                    level=>(
+                                        <Dropdown.Item key={level.id} href={`/books/${key}/${level.name}`}>
```

Comment:
Urls should use `ids` (which are the same in all languages) rather than names (which may be i18n).

Reply:
j'ai essayé de suivre le modèle de "https://www.lelivrescolaire.fr/accueil/matiere/lycee/262/francais", et je n'ai pas fais attention pour le mots "books" qui devrait être remplacé par livres

Reply:
👍 for using the full site as reference. LLS used to have really bad URL design, they upgraded a while ago to make them logical and SEO.